Mathlib/GroupTheory/MonoidLocalization.lean

```diff
@@ -1609,6 +1609,27 @@ theorem of_mulEquivOfMulEquiv {k : LocalizationMap T Q} {j : M ≃* P} (H : S.ma
 #align submonoid.localization_map.of_mul_equiv_of_mul_equiv Submonoid.LocalizationMap.of_mulEquivOfMulEquiv
 #align add_submonoid.localization_map.of_add_equiv_of_add_equiv AddSubmonoid.LocalizationMap.of_addEquivOfAddEquiv
 
+
+@[to_additive]
+theorem injective_iff (f : LocalizationMap S N) :
```

Comment:
```suggestion
theorem injective_toMap {f : LocalizationMap S N} :
```

Reply:
Why do you propose that name? It is a bit confusing for me the names to use.

Reply:
My reasoning here is that:
* You're talking about injectivity of `f.toMap`, so it should be `toMap_injective_something` or `injective_toMap_something` (we tend to prefer the former for predicates like `Injective`, `Surjective`, `Monotone`...)
* It is primarily about `LocalizationMap`, so it should be in namespace `LocalizationMap`
* Actually, `LocalizationMap.toMap_injective` is ambiguous, since it could refer to injectivity of `LocalizationMap.toMap` (instead of `f.toMap`), so probably `toMap_injective_iff` is better.